Code/Framework/AzToolsFramework/AzToolsFramework/ComponentModes/CapsuleComponentMode.cpp

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include "CapsuleComponentMode.h"
+#include <AzToolsFramework/ComponentModes/ShapeTranslationOffsetViewportEdit.h>
```

Comment:
uber nit: Add line break between `""` and `<>` style includes

Reply:
Done